src/cascadia/PublicTerminalCore/HwndTerminal.cpp

```diff
@@ -725,6 +711,10 @@ try
     {
         modifiers |= ControlKeyStates::EnhancedKey;
     }
+    if (vkey && keyDown && _uiaProvider)

```

Comment:
Can there be `_SendKeyEvent`s without `vkey`?

Reply:
Traced all the way to `TerminalContainer.cs`:
`private IntPtr TerminalContainer_MessageHook(IntPtr hwnd, int msg, IntPtr wParam, IntPtr lParam, ref bool handled)`

`wParam` gets converted to the vkey. I assume vkey can't be null? But I guess it doesn't hurt